### ISSUE TITLE
Disable warning about custom eval_metric while loading model

### ISSUE THREAD

**Issue by AndreyGurevich** [catboost/catboost#1169]
I use custom evaluation metric during training: `eval_metric=SacredMetricsSender()`.
After training, I save it to the cbm file and load in another module (Python or Java - we use both).
While loading it prints warning to stdout:
```
Parameter {"eval_metric":"PythonUserDefinedPerObject"} is ignored, because it cannot be parsed.
There are invalid params and some of them will be ignored.
```
Can we disable this output?

**Comment by pplonski** [comment]
I got the same problem, and looks like the same problem is here: https://github.com/catboost/catboost/issues/873

**Comment by kizill** [comment]
Fixed in 0.26.1

**Comment by mazurkin** [comment]
catboost for java 0.26.1:

```
There are invalid params and some of them will be ignored.
Parameter {"feature_weights":[1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1]} is ignored, because it cannot be parsed.
```
